code/global.dm

```diff
@@ -7,7 +7,6 @@
 #define R_BUILDMODE		(1<<0)
 #define R_ADMIN			(1<<1)
 #define R_BAN			(1<<2)
-#define R_FUN			(1<<3)
```

Comment:
shift all the other ones below up one (i.e. R_SERVER (1<<3)

Reply:
There was one number already missing, so I had originally left this alone. Rearranged all of them to be in sequence now.